package.json

```diff
@@ -31,7 +31,7 @@
     "commander": "12.1.0",
     "cors": "2.8.5",
     "deepcopy": "2.1.0",
-    "express": "4.21.0",
+    "express": "^5.0.0",
```

Comment:
Please revert package.json and package-lock.json and then run `npm  i -E express@5.0.1`